{
	"id": "CVE-2020-28424",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-28424",
				"description": "This affects all versions of package s3-kilatstorage.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 9.8,
							"base_severity": "CRITICAL",
							"temporal_score": 9.8,
							"temporal_severity": "CRITICAL",
							"environmental_score": 9.8,
							"environmental_severity": "CRITICAL"
						}
					},
					{
						"type": "cvss_v31",
						"source": "report@snyk.io",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:L/I:L/A:N",
							"base_score": 7.2,
							"base_severity": "HIGH",
							"temporal_score": 7.2,
							"temporal_severity": "HIGH",
							"environmental_score": 7.2,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-78"
						]
					}
				],
				"references": [
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://security.snyk.io/vuln/SNYK-JS-S3KILATSTORAGE-1050396"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-28424"
					},
					{
						"source": "report@snyk.io",
						"url": "https://security.snyk.io/vuln/SNYK-JS-S3KILATSTORAGE-1050396"
					}
				],
				"published": "2022-08-02T14:15:09.527Z",
				"modified": "2024-11-21T05:22:47Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"cpe": "cpe:2.3:a:s3-kilatstorage_project:s3-kilatstorage:*:*:*:*:*:node.js:*:*"
															}
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2020/CVE-2020-28424.json"
		]
	}
}
